index.js

```diff
@@ -22,33 +22,60 @@ exports.register = (server, config, next) => {
   const makeSlackPayload = (tags, data) => {
     let slackPayload = {};
     if (_.isString(data)) {
-      slackPayload = {
-        attachments: [{
-          text: `${data} [${tags}] `
-        }]
-      };
-    } else if (_.isObject(data)) {
-      // if it's a json object then format it so
-      // it displays all nicely on slack:
-      if (!data.message) {
-        // slack uses ``` to format text like an object:
+      if (config.noTags) {
```

Comment:
can you look at making this a little cleaner?  There are a lot of if config.noTags checks.  Maybe make the tags string include the `[ ]` and then if noTags, just set the whole thing to blank string and you can still include it